config/core/test-configs.yaml

```diff
@@ -2305,6 +2305,7 @@ test_configs:
     test_plans:
       - baseline
       - baseline-nfs
+      - kselftest-futex
```

Comment:
This isn't working for some reason:
https://lava.collabora.co.uk/scheduler/job/4246916

Dropping this test plan from the PR for now.